README.md

```diff
@@ -1 +1,93 @@
-# hapi-rate-limit
\ No newline at end of file
+# hapi-rate-limit
+A [Hapi](http://hapijs.com/) plugin that enables rate-limiting for GET, POST, and DELETE requests. This plugin can be configured with custom
+rates on a route-by-route basis.
+
+## Register the plugin
+```
+const Hapi = require('hapi');
+
+const server = new Hapi.Server();
+
+server.register([
+  register: require('../lib'),
```

Comment:
This should be `register: require('hapi-rate-limit'),` to mimic how it'll be used by others.